stan/math/prim/prob.hpp

```diff
@@ -1,326 +1,326 @@
+#include <stan/math/prim/prob/gaussian_dlm_obs_rng.hpp>
+#include <stan/math/prim/prob/inv_wishart_log.hpp>
+#include <stan/math/prim/prob/inv_wishart_lpdf.hpp>
+#include <stan/math/prim/prob/inv_wishart_rng.hpp>
+#include <stan/math/prim/prob/lkj_corr_cholesky_log.hpp>
+#include <stan/math/prim/prob/lkj_corr_cholesky_lpdf.hpp>
+#include <stan/math/prim/prob/lkj_corr_cholesky_rng.hpp>
+#include <stan/math/prim/prob/lkj_corr_log.hpp>
+#include <stan/math/prim/prob/lkj_corr_lpdf.hpp>
+#include <stan/math/prim/prob/lkj_corr_rng.hpp>
+#include <stan/math/prim/prob/lkj_cov_log.hpp>
+#include <stan/math/prim/prob/lkj_cov_lpdf.hpp>
+#include <stan/math/prim/prob/matrix_normal_prec_log.hpp>
+#include <stan/math/prim/prob/matrix_normal_prec_lpdf.hpp>
+#include <stan/math/prim/prob/matrix_normal_prec_rng.hpp>
+#include <stan/math/prim/prob/multi_gp_cholesky_log.hpp>
+#include <stan/math/prim/prob/multi_gp_cholesky_lpdf.hpp>
+#include <stan/math/prim/prob/multi_gp_log.hpp>
+#include <stan/math/prim/prob/multi_gp_lpdf.hpp>
+#include <stan/math/prim/prob/multi_normal_cholesky_log.hpp>
+#include <stan/math/prim/prob/multi_normal_cholesky_lpdf.hpp>
+#include <stan/math/prim/prob/multi_normal_cholesky_rng.hpp>
+#include <stan/math/prim/prob/multi_normal_log.hpp>
+#include <stan/math/prim/prob/multi_normal_lpdf.hpp>
+#include <stan/math/prim/prob/multi_normal_prec_log.hpp>
+#include <stan/math/prim/prob/multi_normal_prec_lpdf.hpp>
+#include <stan/math/prim/prob/multi_normal_prec_rng.hpp>
+#include <stan/math/prim/prob/multi_normal_rng.hpp>
+#include <stan/math/prim/prob/multi_student_t_log.hpp>
+#include <stan/math/prim/prob/multi_student_t_lpdf.hpp>
+#include <stan/math/prim/prob/multi_student_t_rng.hpp>
+#include <stan/math/prim/prob/multinomial_log.hpp>
+#include <stan/math/prim/prob/multinomial_lpmf.hpp>
+#include <stan/math/prim/prob/multinomial_rng.hpp>
+#include <stan/math/prim/prob/neg_binomial_2_log_glm_log.hpp>
+#include <stan/math/prim/prob/neg_binomial_2_log_glm_lpmf.hpp>
+#include <stan/math/prim/prob/normal_id_glm_log.hpp>
+#include <stan/math/prim/prob/normal_id_glm_lpdf.hpp>
+#include <stan/math/prim/prob/ordered_logistic_glm_lpmf.hpp>
+#include <stan/math/prim/prob/ordered_logistic_log.hpp>
+#include <stan/math/prim/prob/ordered_logistic_lpmf.hpp>
+#include <stan/math/prim/prob/ordered_logistic_rng.hpp>
+#include <stan/math/prim/prob/ordered_probit_log.hpp>
+#include <stan/math/prim/prob/ordered_probit_lpmf.hpp>
+#include <stan/math/prim/prob/ordered_probit_rng.hpp>
+#include <stan/math/prim/prob/poisson_log_glm_log.hpp>
+#include <stan/math/prim/prob/poisson_log_glm_lpmf.hpp>
+#include <stan/math/prim/prob/wishart_log.hpp>
+#include <stan/math/prim/prob/wishart_lpdf.hpp>
+#include <stan/math/prim/prob/wishart_rng.hpp>
 
-#include <stan/math/prim/scal/prob/bernoulli_ccdf_log.hpp>
-#include <stan/math/prim/scal/prob/bernoulli_cdf.hpp>
-#include <stan/math/prim/scal/prob/bernoulli_cdf_log.hpp>
```

Comment:
Does it still make sense to keep the distinction between those that were in mat and those that were in scal? I reckon that soon it will not, so I would reorder them alphabetically already.

Reply:
Yeah we should reorder. Done.